### PR TITLE
executor: change ut parameters in parallel aggregate hash spill ut

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What problem does this PR solve?

Issue Number: ref #47733

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/executor/aggregate/agg_spill_test.go [hunk 1]
```diff
@@ -347,8 +347,8 @@ func TestGetCorrectResult(t *testing.T) {
 	ctx := mock.NewContext()
 	initCtx(ctx, newRootExceedAction, hardLimitBytesNum, 256)
 
-	rowNum := 100000 + rand.Intn(100000)
-	ndv := 50000 + rand.Intn(50000)
+	rowNum := 100000
+	ndv := 50000
```
Comment (guo-shaoge) [inline pkg/executor/aggregate/agg_spill_test.go:351]: better disalbe `slowSomePartialWorkers` failpoint when case finish

Comment (guo-shaoge) [reply pkg/executor/aggregate/agg_spill_test.go:351]: reset LGTM

Comment (xzhangxian1008) [reply pkg/executor/aggregate/agg_spill_test.go:351]: > better disable `slowSomePartialWorkers` failpoint when case finish

done

**Review by bb7133 — APPROVED** [review]
LGTM

**Review by guo-shaoge — APPROVED** [review]

**Pull request merged**